scripts/create-component/package-template/.storybook/config.js

```diff
@@ -7,10 +7,10 @@ import themeDecorator from '@pluralsight/ps-design-system-storybook-addon-theme'
 addDecorator(centerDecorator)
 addDecorator(themeDecorator(addons))
 
-const req = require.context('../stories', true, /\.story\.js$/)
+const req = require.context('../src/react', true, /\.story\.js$/)
```

Comment:
does this allow __stories__ subdirs?  If we can't get to that, I think I prefer keeping packages/X/stories dirs.

Reply:
yeah it will find any .story file in src/react